src/dmd/statementsem.d

```diff
@@ -3922,9 +3922,9 @@ package (dmd) extern (C++) final class StatementSemanticVisitor : Visitor
         // use setImpure/setGC when the deprecation cycle is over
```

Comment:
As the comments says, instead of raising an error, you can just set the function to be impure / gc

Reply:
Ah right. Thanks, going to fix it :)